src/constants/config.ts

```diff
@@ -3,8 +3,6 @@ import { SdkConfig } from '../types'
 export const DEFAULT_SDK_CONFIG: SdkConfig = {
   cacheIdentity: true,
   trackPages: true,
-  trackReferrer: true,
-  trackUTM: true,
```

Comment:
SDK installers can no longer opt-out of UTM and referrer tracking 🥇 Please correct me only if that's wrong